validator-api/src/node_status_api/routes.rs

```diff
@@ -281,6 +281,7 @@ pub(crate) async fn get_mixnode_avg_uptime(
     }))
 }
 
+// DEPRECATED: the uptime is available as part of the `/mixnodes/detailed` endpoint
```

Comment:
there's a `#[deprecated]` annotation btw

Reply:
Nice, I'll need to remove all calls to this from the explorer-api first though